src/pkg/caendr/caendr/services/sql/etl/strains.py

```diff
@@ -4,18 +4,21 @@
 from caendr.services.elevation import get_elevation
 from caendr.services.cloud.sheets import get_google_sheet
 from caendr.services.cloud.secret import get_secret
+from caendr.models.datastore import SPECIES_LIST
 from caendr.models.sql import Strain
 
+# Get list of Google Sheet IDs for each species
+# Expects secret names with prefix "ANDERSEN_LAB_STRAIN_SHEET_"
+# and species ID in all caps
 ANDERSEN_LAB_STRAIN_SHEETS = [
-  get_secret('ANDERSEN_LAB_STRAIN_SHEET'),            # elegans
-  get_secret('ANDERSEN_LAB_STRAIN_SHEET_C_BRIGGSAE'), # briggsae
-  # tropicalis
+  get_secret(f'ANDERSEN_LAB_STRAIN_SHEET_{species_name.upper()}')
```

Comment:
@vlagrassa we should have a list of 'allowed' species names that can be read from the ENV (i.e. validate the species name before looking it up).

Reply:
Normally I'd check that by making sure the `species_name` is in `SPECIES_LIST`, but here I'm generating it from the species list itself.  Should I check to make sure the env var exists before trying to get it, maybe?

Alternatively, I could go back to hard-coding the secret names.  It might not make sense to generate them dynamically, now that I think about it.